Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,35 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
-
 // Add your code here
+// Map the tasks to durations in hours
+
+const durationInMinute = [];
+for (let k in tasks) {
+  durationInMinute[k] = tasks[k].duration;
+}
+
+let durationInHours = durationInMinute.map(i => (Number((i / 60).toFixed(2))));
+console.log(durationInHours);
+
+[3, 2, 0.33, 3.33, 4, 3, 0.17, 3.33, 0.67]
```

Comment:
when ever you wanna show the expected result or you wanna show that this is result from that, it should be in a comment not on a new line without anything.

remember the shortcut?